go/interpreter/lfvm/interpreter.go

```diff
@@ -190,7 +190,7 @@ func getOutput(ctxt *context) ([]byte, error) {
 				return nil, err
 			}
 			res = make([]byte, size)
-			ctxt.memory.CopyData(offset, res)
+			ctxt.memory.copyData(offset, res)
```

Comment:
Can't this be a `getSlice` and the `copyData` can be dropped?

Reply:
well, it would have be a `copy(getSlice(), res)` ... yes.